content/docs/cml/self-hosted-runners.md

```diff
@@ -83,27 +83,27 @@ you do it your way :)
-| `-h`                     | Show help                                                                                                       | Boolean                                                                                                        |
+| Argument                 | Type                                                                  | Default                                                    | Description                                                                                                                  |
+| ------------------------ | --------------------------------------------------------------------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Flag                                                                  | Unset                                                      | Show version number and exit.                                                                                                |
+| `--labels`               | List                                                                  | `cml`                                                      | Comma delimited list of runner labels.                                                                                       |
+| `--idle-timeout `        | Integer                                                               | `300`                                                      | Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to `0` disables automatic shutdown.   |
+| `--name`                 | String                                                                | `cml-{identifier}` where `{identifier}` is a random string | Name displayed in the repository once registered.                                                                            |
+| `--single`               | Flag                                                                  | Unset                                                      | Exit after running a single job.                                                                                             |
+| `--reuse`                | Flag                                                                  | Unset                                                      | Don't launch a new runner if an existing one has the same name or overlapping labels.                                        |
+| `--driver`               | Choice: `github` or `gitlab`                                          | Inferred from environment variables                        | Driver for the platform where the repository is hosted.                                                                      |
```

Comment:
Another example on how to put type into Description when needed:

```suggestion
| `--driver`               | Inferred from environment variables                        | Platform where the repository is hosted. Accepts `github` or `gitlab`.                                                                      |
```

Reply:
Resolved with https://github.com/iterative/dvc.org/pull/2321/commits/ca804af226acd6100e23d25430d39b87277ef6f8